rsmtool/preprocessor.py

```diff
@@ -2446,7 +2386,7 @@ def process_data_rsmpredict(self, config_obj, data_container_obj):
 
         # if we are using a newly trained model, use trim_tolerance from the
```

Comment:
I kept this as is because sometimes we want to use very old models to produce scores.